msteams-platform/toolkit/explore-Teams-Toolkit.md

```diff
@@ -56,7 +53,7 @@ From the task pane, you can see the following sections:
 
```

Comment:
Teams Toolkit image(../assets/images/teams-toolkit-v2/teams-toolkit-sidebar-icon.PNG) is not aligned with other icons. It's not selected so that the color is grey.

Reply:
Resolved